token-lending/sdk/src/state/reserve.rs

```diff
@@ -28,6 +28,12 @@ pub const LIQUIDATION_CLOSE_AMOUNT: u64 = 2;
 /// Maximum quote currency value that can be liquidated in 1 liquidate_obligation call
 pub const MAX_LIQUIDATABLE_VALUE_AT_ONCE: u64 = 500_000;
 
+/// Maximum bonus received during liquidation. includes protocol fee.
+pub const MAX_BONUS_PCT: u8 = 25;
+
+/// Maximum protocol liquidation fee
+pub const MAX_PROTOCOL_LIQUIDATION_FEE: u8 = 5;
```

Comment:
oh this is very different than what i envisioned

Reply:
which part? how the protocol liquidation fee is calculated?

Reply:
next time on...

Reply:
but fr can we get a way to be a bit more granular here? maybe basis points or deca-basispoints.